test/unit/BNum.tree

```diff
@@ -93,5 +93,5 @@ BNumTest::bpow
 │   └── it should revert
 ├── when passing base gte MAX_BPOW_BASE
 │   └── it should revert
-└── when passing known values // generic for some given values
+└── when passing known values
```

Comment:
opinions on hard-coding the values themselves here as well?

Reply:
mhmmm too much noise? i think we need to leave the tree with as less implementation details as possible